### PR TITLE
Resolves #1970 script reads entire date/time now

### LINKED ISSUE #1970: BUG: DAIL - SEND NOMI not entering dates right
I was running the dail scrubber on my revs scrubber messages (granted I don't remember if we used a local version of revs scrubber) and the client didn't turn in CAF so I selected don't sent nomi option. It entered this as the case note.

2016-03-09 10_03_20-s1 - anoka desktop - anoka scripted desktop - bluezone mainframe display

Here is the dail:

dail

### PR DESCRIPTION
**Pull request by CDPotter**
BLIP: Changed way script reads TIKL by just reading entire date and time in one line and making it one variable.

Figured that reading the entire rest of the sentence would make sense. I also accounted for an extra 3 digits for MM DD and HH

### CODE REVIEW

File: Script Files/DAIL/DAIL - SEND NOMI.vbs [hunk 1]
```diff
@@ -83,9 +83,9 @@ EMConnect ""
 EMSendKey "x"
 transmit
 
-'Reading date and time of recertification appointment from the TIKL--DAIL message that should be read is: "~*~*~CLIENT HAD RECERT INTERVIEW AT..." This is the part that is static in the DAIL message
-EMReadScreen interview_date, 10, 9, 42
-EMReadScreen interview_time, 8, 9, 56
+'Reading date and time of recertification appointment from the TIKL--DAIL message that should be read is: "~*~*~CLIENT HAD RECERT INTERVIEW APPT AT..." This is the part that is static in the DAIL message
+EMReadScreen interview_date_time, 19, 9, 46
+'EMReadScreen interview_time, 8, 9, 56
```
Comment (theVKC) [inline Script Files/DAIL/DAIL - SEND NOMI.vbs]: Can we clear this out?

File: Script Files/DAIL/DAIL - SEND NOMI.vbs [hunk 2]
```diff
@@ -83,9 +83,9 @@ EMConnect ""
 EMSendKey "x"
 transmit
 
-'Reading date and time of recertification appointment from the TIKL--DAIL message that should be read is: "~*~*~CLIENT HAD RECERT INTERVIEW AT..." This is the part that is static in the DAIL message
-EMReadScreen interview_date, 10, 9, 42
-EMReadScreen interview_time, 8, 9, 56
+'Reading date and time of recertification appointment from the TIKL--DAIL message that should be read is: "~*~*~CLIENT HAD RECERT INTERVIEW APPT AT..." This is the part that is static in the DAIL message
+EMReadScreen interview_date_time, 19, 9, 46
```
Comment (theVKC) [inline Script Files/DAIL/DAIL - SEND NOMI.vbs]: I like this solution! But, is it possible to trip it up with a REALLY long date and time? Like: "10/10/2016 at 12:45 PM"? I'd bet that's the char maximum... does the TIKL overflow in these instances?

**Comment by theVKC** [comment]
See comments, @CDPotter. Thanks!

File: Script Files/DAIL/DAIL - SEND NOMI.vbs (hunk 2 above)
Comment (CDPotter) [inline Script Files/DAIL/DAIL - SEND NOMI.vbs]: I made it 19 to account for those. The TIKL is exactly long enough to handle it that full date.

2016-03-10 08_55_41-s1 - anoka desktop - anoka scripted desktop - bluezone mainframe display

"MM/DD/YYYY HH:MM PM)

**Comment by CDPotter** [comment]
Fixed em.

**Pull request merged**